.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,165 @@
+﻿# Contributing guidelines
+
+## Table of Contents
+
+- [Getting started](#getting-started)
+  - [Language](#language)
+    - [For native English speakers](#for-native-english-speakers)
+  - [Code of Conduct](#code-of-conduct)
+- [How can I contribute?](#how-can-i-contribute)
+  - [Documentation](#documentation)
+  - [Issues](#issues)
+    - [Submitting an issue](#submitting-an-issue)
+  - [Feedback](#feedback)
+  - [Code](#Code)
+    - [Dev environment](#dev-environment)
+- [Commiting](#Commiting)
+- [Submitting a pull request](#submitting-a-pull-request)
+
+## Getting started
+
+First off, we would like to thank you for taking the time to contribute and make this a better project!
+
+Here we have a set of instructions and guidelines to reduce misunderstandings and make the process of contributing to `unform` as smooth as possible.
+
+We hope this guide makes the contribution process clear and answers any questions you may have.
+
+### Language
+
+Please, while contributing or interacting in any way in this project, refrain from using any language other than **English**.
+
+#### For native English speakers
+
+Try to use simple words and sentences. Don't make fun of non-native English speakers if you find something wrong about the way they express themselves
+
+Try to encourage newcomers to express their opinions, and make them comfortable enough to do so.
+
+### Code of Conduct
+
+We expect that project participants to adhere to our Code of Conduct. You can check the the [full text]](CODE_OF_CONDUCT.md) so that you may understand the kind of conduct we are expecting and what actions will and will not be tolerated.
+
+By participating in this project, you agree to abide by its terms.
+
+## How can I help?
+
+Here are some ways you can help along with some guidelines.
+
+### Documentation
+
+As a user of `unform`, you're the perfect candidate to help us improve our documentation!
+
+Typos, errors, lack of examples and/or explanation, etc. This are some examples of for things that could be fixed and/or improved.
+
+You could even make improvements to this guide! :)
+
+While documenting, try to keep things simple and clear.
+
+### Issues
+
+Some issues are created with missing information, without a template, not reproducible, or plain
```

Comment:
Do we have issue templates already?

Reply:
I'm working on it in this exact moment. 😃